Specifications/Compiler/com.apple.compilers.model.coredata.xcspec

```diff
@@ -0,0 +1,74 @@
+/**
+ Copyright (c) 2015-present, Facebook, Inc.
+ All rights reserved.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree. An additional grant
+ of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+(
+    {
+        Type = Compiler;
+        Identifier = com.apple.compilers.model.coredata;
+        Name = "Data Model Compiler (momc)";
+
+        CommandLine = "momc [options] $(InputFile) $(ProductResourcesDir)/";
```

Comment:
nit: Replace `$(InputFile)` with `$(InputPath)` — standardizing on that for all the specifications since there's no reason for multiple.